packages/scene-composer/src/components/panels/scene-settings/MatterportTagSync.tsx

```diff
@@ -156,8 +155,12 @@ export const MatterportTagSync: React.FC = () => {
         size='small'
         triggerType='custom'
         content={
-          <StatusIndicator type='success'>
-            {intl.formatMessage({ defaultMessage: 'Tags sync successful', description: 'Status indicator label' })}
+          <StatusIndicator type={syncTagStatus} data-testid='sync-button-status'>
```

Comment:
aria-label for statusIndicator?

Reply:
it shows icon + text, no need to add the label

Reply:
You can wrap this inside a div and have an aria-label if we want to support accessibility if polaris component does not support directly.